blob-tool/src/main.rs

```diff
@@ -24,39 +25,42 @@ async fn main() {
         .await
         .expect("Failed creating celestia rpc client");
 
-    let header = client.header_get_by_height(args.height)
+    let header = client
+        .header_get_by_height(args.height)
         .await
         .expect("Failed getting header");
 
     let commitment = Commitment::new(
-        hex::decode(&args.commitment)
-            .expect("Invalid commitment hex")
+        base64::engine::general_purpose::STANDARD
```

Comment:
can you make it do either hex or base64? Maybe try hex first and if it fails try base64?